system_tests/extnet_tests.py

```diff
@@ -51,6 +51,8 @@ class ExtNetTest(BaseTestCase):
     _ip_range1 = '10.10.30.2-10.10.30.99'
     _ip_range2 = '10.10.30.30-10.10.30.60'
     _ip_range3 = '10.10.30.101-10.10.30.110'
+    _vc_name = 'vc2'
+    _pg_name = 'DPortGroup'
```

Comment:
Copy the "PortgroupHelper" under pysdk/system-tests/ to vcd-cli/system-tests... call the platform.getmethod... to get the portgroup